packages/cardpay-cli/scheduled-payment/enable-module.ts

```diff
@@ -38,6 +38,6 @@ export default {
     );
     console.log(`scheduled payment module deployed to: ${scheduledPaymentModuleAddress}`);
```

Comment:
It would look better if the capitalization of the sentences in console log outputs was consistent

Reply:
updated